packages/connect/CHANGELOG.md

```diff
@@ -1,12 +1,12 @@
-|             Package              | Stable | Canary |
-| :------------------------------: | :----: | :----: |
-|       npm @trezor/connect        | 9.3.0  |   -    |
-|     npm @trezor/connect-web      | 9.3.0  |   -    |
-| npm @trezor/connect-webextension | 9.3.0  |   -    |
-
-|     Deployment     | Stable | Canary |
-| :----------------: | :----: | :----: |
-| connect.trezor.io/ | 9.3.0  |   -    |
+|             Package              | Stable |    Canary    |
+| :------------------------------: | :----: | :----------: |
+|       npm @trezor/connect        | 9.3.0  | 9.3.1-beta.1 |
+|     npm @trezor/connect-web      | 9.3.0  | 9.3.1-beta.1 |
+| npm @trezor/connect-webextension | 9.3.0  | 9.3.1-beta.1 |
+
+|     Deployment     | Stable |    Canary    |
+| :----------------: | :----: | :----------: |
+| connect.trezor.io/ | 9.3.0  | 9.3.1-beta.1 |
 
 Use the persistent link [connect.trezor.io/9](https://connect.trezor.io/9/) to access the latest stable version of Connect Explorer.
```

Comment:
yes, no changelog here.

Reply:
it is not intended, right? There should be a changelog, no?

Reply:
not intended, something is broken

Reply:
Actually it is intended, the connect changelog is added manually.
We have automation for all the connect packages to create the changelog automatically but for connect since we want to have a more human friendly / curated changelog is a manual process. I is always have been like that.

Reply:
Ok, maybe we were just too fast. 😄

Reply:
Ok, thats right. I thought it creates some barebones of changelog and it is only edited later. It got somehow mixed up in my head. sry

Reply:
Yes, you are right with the barebones of changelog that you mentioned is added as a comment to this PR https://github.com/trezor/trezor-suite/pull/13480#issuecomment-2245440268